app/src/main/java/com/greenaddress/greenbits/ui/transactions/MainFragment.java

```diff
@@ -127,13 +132,16 @@ public View onCreateView(final LayoutInflater inflater, final ViewGroup containe
 
         mAssetsSelection = UI.find(mView, R.id.assetsSelection);
         mAssetsSelection.setOnClickListener(v -> startActivityForResult(new Intent(getGaActivity(),
-                                                                                   AssetsSelectActivity.class),
-                                                                        REQUEST_SELECT_ASSET));
+                        AssetsSelectActivity.class),
+                REQUEST_SELECT_ASSET));
         mSwitchNetwork = UI.find(mView, R.id.switchNetwork);
         mSwitchNetwork.setOnClickListener(v -> showDialog());
         mSwitchNetwork.setText(getNetwork().getName());
         mSwitchNetwork.setTextColor(getResources().getColor(R.color.white));
 
+        boolean scrollEnable = getSharePreferenceActivity().getBoolean(PrefKeys.MAIN_FRAGMENT_ID_FIX, true);
+        changeBehaviorScrollView(scrollEnable);
```

Comment:
It is not clear to me why you need to store `scrollEnable` in shared preference.
`showTxView()` updates scrollview a bit down in the code. If you are looking for a default value, I think you could set programmatically because the previous value could be a wrong in some case (change subaccount, change wallet).

Reply:
Hi @lvaccaro, 

I agree with you, this is my fault, I used my logic inside the commit 1 also in the commit 2. I can remove the share preference and also I can remove the local variable. I can call the method `changeBehaviorScrollView` only inside the `showTxView`. My logic now makes sense?

Reply:
Yes, concept ack.
I don't know if you need some sort of default behaviour until the txs are not loaded yet, but this should not be so relevant.

Reply:
>I don't know if you need some sort of default behaviour until the txs are not loaded yet, but this should not be so relevant.

Oh, at the moment no, because I want to know only when the list of the transaction is empty, and it is empty when the label is visible. I think that the solution with your feature worked. I already tested